toolbar_utils.py

```diff
@@ -106,7 +106,7 @@ def radio_factory(name, toolbar, callback, cb_arg=None, tooltip=None,
 def label_factory(toolbar, label_text, width=None):
     ''' Factory for adding a label to a toolbar '''
     label = Gtk.Label(label_text)
-    label.set_line_wrap(True)
+    label.set_line_wrap(False)
```

Comment:
Can you explain me the change necessary here?

Reply:
This change did fix the toolbar vertical expansion problem.  It may risk horizontal expansion instead on small screens.  :grin:

Reply:
Nice :joy:

Reply:
I reduced the length of the text, so hopefully there won't be an issue of horizontal expansion. Done in commit `766b774`